docs/unified-test-documentation/dasharo-compatibility/312-nvme-support.md

```diff
@@ -95,14 +95,13 @@ M.2 slot.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
-1. Open device manager.
-1. Select `Disk drives` option and find correct drive.
-1. Right click the drive entry and select `Properties` option
-1. Go to `Details` tab.
-1. In the `Property` menu select `Hardware Ids` option.
-1. Find the `Value` window and note the result.
+1. Open PowerShell as administrator.
+1. Run below command and note result:
+
+    ```powershell
+    Get-PnpDevice -Status "OK" | where { $_.InstanceId -like "SCSI\DISK&VEN_NVME&*"}
+    ```
 
 **Expected result**
 
-1. The `OPERATING_SYSTEM` booting from the NVMe disk
```

Comment:
I think, that we should leave the first point and in the second one show the sample output.